apps/internal/local/server.go

```diff
@@ -142,11 +162,19 @@ func (s *Server) handler(w http.ResponseWriter, r *http.Request) {
 
 	headerErr := q.Get("error")
 	if headerErr != "" {
-		desc := html.EscapeString(q.Get("error_description"))
```

Comment:
Since this line was removed, besides I am not seeing similar escaping on `error`, can we have a unit test case to ensure the escaping happens?